docs/iot/iot-partners/index.md

```diff
@@ -0,0 +1,26 @@
+# Tizen IoT x Nubison Cloud
+
+Nubison cloud can readily be interconnected with Tizen IoT devices.
+Using Nubison cloud, users can easily send data or receive requests for Tizen service applications on real-time basis.
+
+**Figure: Architecture**
+![Architecture](media/architecture.png)
+
+Compatibility
+- Platform : Tizen 5.0 IoT Headed & Headless and later
+- SDK : Tizen Studio 3.0 and later 
+
+
+## What is Nubison IoT?
+Nubison IoT, as an IoT service provider, lets anyone to easily use various features and services, inclduing data analysis function, specialized for IoT environment.
```

Comment:
Nubison IoT, is an IoT service provider. It provides various IoT features and services which can be utilized in various IoT devices, one such a distinguished service is: data analysis service. It is a specialized service developed specifically for IoT environments.

Reply:
done :)